CHANGELOG.md

```diff
@@ -90,7 +106,7 @@ TODO: `cargo`, `std` features
     * `App::help_message` in favor of `App::mut_arg`
     * `App::help_short` in favor of `App::mut_arg`
     * `App::arg_from_usage` in favor of `App::arg`
-    * `App::args_from_usage` in favor of TODO:
+    * `App::args_from_usage` in favor of TODO: `App::args(usage.split('\n'))`
```

Comment:
Is this TODO redundant now or is this line still missing some information?

```suggestion
    * `App::args_from_usage` in favor of `App::args(usage.split('\n'))`
```

Reply:
I have yet the test the alternative. So, have the `TODO` there to remind me.